QUARKUS-1414.md

```diff
@@ -0,0 +1,35 @@
+# QUARKUS-1414 Service Binding improvements
+
+Jira: https://issues.redhat.com/browse/QUARKUS-1414
+
+The goal of this feature is to improve the provided technology preview support of OpenShift Service Binding in Quarkus
+applications.
+
+## Scope of the testing
+
+### Impact on test suites and testing automation
+* We need to ensure all the supported means of producing Quarkus deployments in OpenShift work with service binding
+  * Source S2I
+  * Docker build
+  * Binary S2I
+* Orchestration of OSD will have to be automated to ensure easy testing on OSD with RHOSAK service bindings. This, as of
```

Comment:
Pleas provide info why OSD is needed and we can't use OCP instances in our lab.

Reply:
This is the venue we've explored in 2.2, according to docs at that time. There is a RHOAS operator that is installable on any cluster, but I'd have to commit more to seeing how that works.